include/oneapi/mkl/detail/backends.hpp

```diff
@@ -46,7 +47,8 @@ static backendmap backend_map = {
     { backend::cublas, "cublas" },       { backend::cusolver, "cusolver" },
     { backend::curand, "curand" },       { backend::netlib, "netlib" },
     { backend::rocblas, "rocblas" },     { backend::rocrand, "rocrand" },
-    { backend::rocsolver, "rocsolver" }, { backend::unsupported, "unsupported" }
+    { backend::rocsolver, "rocsolver" }, { backend::unsupported, "unsupported" },
+    { backend::syclblas, "syclblas" },   { backend::unsupported, "unsupported" }
```

Comment:
I've noticed it only now, we shouldn't be adding the `unsupported` backend twice here.

Reply:
Done in [75e7e6e](https://github.com/oneapi-src/oneMKL/pull/262/commits/75e7e6ef3204543f1818b92d7ec0ebeee52b2f53).